lib/sqlalchemy/dialects/postgresql/psycopg2.py

```diff
@@ -610,6 +691,8 @@ def __init__(
         client_encoding=None,
         use_native_hstore=True,
         use_native_uuid=True,
+        execution_mode=None,
+        page_size=10000,
```

Comment:
these names are both too generic if they are at the dialect level.    execution_mode needs to refer to "executemany" and "page_size" needs to refer to "execute_values".

Reply:
OK.
page_size can be used for execute_batch as well. I didn't use it in this pull request to keep the notion that no change should be made when not using the new flags. However, it will improve the performance considerably. So, if we want to add it to the call to execute_batch(), maybe we should give the flag a more generic name (like executemany_page_size) ?

Reply:
just that it has "executemany" in the name should be enough.

Reply:
ok